lib/db/data.js

```
@@ -13,24 +13,37 @@ const db = (connection) => {
   db.getPrompts = query('SELECT * FROM prompts');
   db.getPromptByValue = query('SELECT * FROM prompts WHERE value = ?');
   db.getActions = query('SELECT * FROM actions');
+  db.getClientActions = query('SELECT * FROM client_actions WHERE clientid = ?');
+  db.deleteClientActions = query('DELETE FROM client_actions WHERE clientid = ?');
+
+  // config needs table, parentKey, dataSet and cb
+  const addSet = (err, rows, config) => {
+    if (err) throw err;
+    const parentId = rows[0].id;
```

what would be best to do here? Should we throw a custom error with something like 'no parent set to add to'?